setup.cfg

```diff
@@ -8,7 +8,6 @@ classifiers =
     Operating System :: OS Independent
     Programming Language :: Python
     Programming Language :: Python :: 3
-    Programming Language :: Python :: 3.7
```

Comment:
I saw `python_requires = >=3.8` a few lines down, which is why I'm cleaning this marker while I'm in there